watertap/costing/watertap_costing_package.py

```diff
@@ -59,6 +60,11 @@ class MixerType(StrEnum):
     CaOH2 = "CaOH2"
 
 
+# # TODO: Revisit types of costs for different options
+class ElectrodialysisCostType(StrEnum):
+    default = "default"

```

Comment:
Do we anticipate creating more things here? Otherwise it is an unnecessary complication.